pkgs/tools/package-management/nvd/default.nix

```
@@ -1,19 +1,28 @@
-{ fetchFromGitLab, installShellFiles, lib, python3, stdenv }:
+{ fetchFromGitLab
+, installShellFiles
+, lib
+, python3
+, stdenv
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "nvd";
   version = "0.2.3";
 
   src = fetchFromGitLab {
     owner = "khumba";
     repo = pname;
```

```suggestion
    repo = finalAttrs.pname;
```

I guess it should be like that

---

Nice catch, :) I missed that obvious one!

---

Actually, using `repo = finalAttrs.pname;` is not a good practice, I updated it in my last commit.